modules/install-acs-operator.adoc

```diff
@@ -14,10 +14,23 @@ Using the OperatorHub provided with {ocp} is the easiest way to install {product
 
 .Procedure
 . Navigate in the web console to the *Operators* -> *OperatorHub* page.
+
 . Enter *Advanced Cluster Security* into the *Filter by keyword* box to find the {product-title} Operator.
+
 . Select the *{product-title} Operator* to view the details page.
+
 . Read the information about the Operator and click *Install*.
-. On the *Install Operator* page, keep the default values for *Update channel*, *Installation mode*, *Installed namespace*, and *Approval strategy*.
+
+. On the *Install Operator* page:
+
+** keep the default value for *Installation mode* as *All namespaces on the cluster*.
+** for *Installed namespace* choose a specific namespace in which to install the Operator. Red Hat recommends installing the {product-title} Operator in the *rhacs-operators* namespace.
```

Comment:
For parallelism between bullets, consider starting bullets with the action: "Choose a specific..."